runtime/near-vm-logic/src/logic.rs

```diff
@@ -23,7 +23,7 @@ pub struct VMLogic<'a> {
     /// receipts creation.
     ext: &'a mut dyn External,
     /// Part of Context API and Economics API that was extracted from the receipt.
-    context: VMContext,
+    pub context: VMContext,
```

Comment:
why is this necessary?

Reply:
Oh, not needed at all, just debugging leftover, thanks for spotting.